documentation/source/users/rmg/input.rst

```diff
@@ -230,8 +230,64 @@ The following is an example of a typical species item, based on methane using SM
 			"""
 	)
 
-.. _reactionsystem:
+.. _forbidden_structures:
+
+Forbidden Structures
+=====================
+
+RMG exlores a wide varitey of structures as a builds a mechanism.
+Sometimes, RMG makes nonsensical, unphysical, or unstable species that make their way into the core.
+
+Luckily, RMG has a remedy to prevent these undesirable structures from making their way into your mechanism.
+RMG's ForbiddenStructures database contains structures that are forbidden from entering the core.
+
+While this database forbids many undesirable structures, it may not contain a species
+or structure you would like to exclude from your model. If you would like to add a forbidden structure for your RMG job,
+you can do so by adding it to your input file in the forbidden block.
+
+The label and structure of each forbidden structure must be specified.
+The structure of the forbidden structure can be defined using either SMILES or
+:ref:`adjacencyList <rmgpy.molecule.adjlist>`.
+
+The following is an example of a forbidden structure item, based on bidentate CO2 using SMILES or adjacency list to define the structure::
+
+	forbidden(
+		label='CO2_bidentate',
+		structure=SMILES("O=C(*)O*"),
+	)
+
+	forbidden(
+		label='CO2_bidentate',
+		structure=adjacencyList(
+			"""
+			1 O u0 p2 c0 {2,D}
+			2 C u0 p0 c0 {1,D} {3,S} {4,S}
+			3 X u0 p0 c0 {2,S}
+			4 O u0 p2 c0 {2,S} {5,S}
+			5 X u0 p0 c0 {4,S}
+			"""
+		)
+	)
 
+If you would like to forbid a functional group from your mechanism, use the ``adjacencyListGroup`` syntax in the forbidden block.
+
+The following is an example of a forbidden structure using ``adjacencyListGroup``::
```

Comment:
I first thought this was an adsorbed CO molecule, and took a long time (of writing and deleting comments) before realizing it was R2C*-O*. Not sure how to fix. Maybe change its label? and/or describe it when saying "..an example of a forbidden structure.."  ?

Reply:
....and now I see you have that description just below on line 288. OK, so just move that up to before the example.